grammars/ruby.cson

```diff
@@ -355,13 +355,13 @@
         (\\.(?![^[:space:][:digit:]])(?>_?\\d)*)?   # fractional part
         ([eE][-+]?\\d(?>_?\\d)*)?                   # 1.23e-4
         |
-        0[xX]\\h(?>_?\\h)*
-        |
-        0[oO]?[0-7](?>_?[0-7])*
-        |
-        0[bB][01](?>_?[01])*
-        |
-        0[dD]\\d(?>_?\\d)*                          # A base indicator can only be used with an integer
+        0
+        (?:
+          [xX]\\h(?>_?\\h)*|
```

Comment:
In this case, each line in the group is an alternative, so I chose to put the `|` at the end. 😉  I'll be consistent, promise.